brave-instrumentation/aws-java-sdk-v2-core/src/test/java/brave/instrumentation/awsv2/ITTracingExecutionInterceptor.java

```diff
@@ -124,68 +145,25 @@
     });
   }
 
-  /*
-   * Tests overriden due to the multi-span implementation of the client, application and client
-   */
-
-  @Override public void usesParentFromInvocationTime() throws Exception {
```

Comment:
this is an async test, so if we have to redo it, we'll need to do this too in another class.